pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.camunda</groupId>
     <artifactId>camunda-release-parent</artifactId>
-    <version>3.3</version>
+    <version>3.7</version>
```

Comment:
why such a bump?

Reply:
3.7 is the latest version - if it works, why not use it?